### PR TITLE
Update login page to UI v3

### PR DESCRIPTION
**Pull request by vpchung**
This PR includes the styling for the login page.

### CODE REVIEW

**Comment by vpchung** [comment]
Screenshot of login page:

Screen Shot 2022-06-21 at 11 14 23 PM

File: libs/challenge-registry/ui/src/_lib-theme.scss [hunk 1]
```diff
@@ -1,9 +1,34 @@
+@use '@angular/material' as mat;
+
+@use './lib/palette' as pal;
 @use './lib/button-github/button-github-theme' as button-github;
 @use './lib/footer/footer-theme' as footer;
 @use './lib/navbar/navbar-theme' as navbar;
 
+@include mat.core();
+@import url('https://fonts.googleapis.com/css2?family=Fira+Sans:wght@300,500&family=Poppins:wght@500,600&display=swap');
```
Comment (tschaffter) [inline libs/challenge-registry/ui/src/_lib-theme.scss:9]: I get the following error in FF:

```console
The resource from “https://fonts.googleapis.com/css2?family=Fira+Sans:wght@300,500&family=Poppins:wght@500&display=swap” was blocked due to MIME type (“text/html”) mismatch (X-Content-Type-Options: nosniff).
```

[Learn more](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options)

image

Do you also have this issue?

Also curious to know whether using `@import` is (still?) the recommended way to import a font (now that we are importing other files with `@use`).

**Comment by tschaffter** [comment]
@rrchai (Reminder) If you hit a blank page like I did, make sure you have initialize Keycloak database using the command from the README.

File: libs/challenge-registry/ui/src/_lib-theme.scss (hunk 1 above)
Comment (vpchung) [reply libs/challenge-registry/ui/src/_lib-theme.scss:9]: Hm, weird!  I have not encountered this error before.

And yes, as mentioned in the footer PR, `@import` is still the method to import a font.  `@use` is to replace Sass' usage of [`@import`](https://sass-lang.com/documentation/at-rules/import), not [CSS'](https://developer.mozilla.org/en-US/docs/Web/CSS/@import).

File: libs/challenge-registry/ui/src/lib/footer/_footer-theme.scss [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500&display=swap');
 
 @mixin color($theme) {
   $config: mat.get-color-config($theme);
-  $primary: map.get($config, primary);
-  $accent: map.get($config, accent);
-  $warn: map.get($config, warn);
-  $background: map.get($config, background);
-  $foreground: map.get($config, foreground);
+  $primary: map.get($config, 'primary');
```
[the remark this answers is not in the document]
Comment (vpchung) [reply libs/challenge-registry/ui/src/lib/footer/_footer-theme.scss:8]: Oh yes, similar to the [navbar PR](https://github.com/Sage-Bionetworks/challenge-registry/pull/353#issuecomment-1162159589), this branch also builds off of `footer-component` (as well as `navbar-component`)

**Pull request merged**